### PR TITLE
Add secondary prefix for selecting radio channel, '.'

### PR DESCRIPTION
**Pull request by HerCoyote23**
## About the PR

Currently if you want to speak on your departmental channel, you type ':X supersecret message'. I have a few gripes with this:
--You have to hold shift to type ':', depending on how you type it, you might end up moving your hands more. It'll probably take longer too.
--If you mess up holding shift, it defaults to ';', which is the Common channel. Now you've sent your supersecret message for all the station to see.   ";t any other traitors out there?"
--It's just overall slower and clunkier, especially if you want to get a quick message out in combat, call for help or the like.

This PR would add the ability to select it with period, as well. People who prefer ':' can still use it if they want.

Unfortunately, '.' is actually already taken. It's used for forcing local chat, so you could say things like ".; testmessage" and have it go to local chat. Not useful except in super specific situations, and even then you can get by without it. 
So this also moves LocalPrefix to >, as that's just shift+'.'
**Media**

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: You can now speak on your department's radio channel using .X

### CODE REVIEW

**Comment by HerCoyote23** [comment]
Eh nevermind, this is probably the wrong channel to ask whether the idea would be accepted. I'll reopen this if I can get confirmation that people would be okay with it.

File: Content.Shared/Chat/SharedChatSystem.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ public override void Shutdown()
             return true;
         }
 
-        if (!input.StartsWith(RadioChannelPrefix))
+        if (!(input.StartsWith(RadioChannelPrefix) || input.StartsWith(RadioChannelAltPrefix)))
```
Comment (ficcialfaint) [inline Content.Shared/Chat/SharedChatSystem.cs:92]: doesn't it work without first bracket you added?

Comment (HerCoyote23) [reply Content.Shared/Chat/SharedChatSystem.cs:92]: Probably, I just have a habit of using brackets liberally. I'll remove it.

Comment (HerCoyote23) [reply Content.Shared/Chat/SharedChatSystem.cs:92]: Actually it doesn't, the ! only applies to the first input.StartsWith. I'll leave the brackets there.

Comment (ficcialfaint) [reply Content.Shared/Chat/SharedChatSystem.cs:92]: > Actually it doesn't, the ! only applies to the first input.StartsWith. I'll leave the brackets there.

Ah, I read the code wrongly mb

**Pull request merged**